src/components/DocumentCard/DocumentCard.scss

```diff
@@ -43,20 +43,27 @@
   // Remove the usual accent color from the preview
   .ms-DocumentCardPreview {
     border-bottom: none;
-
-    .ms-Image {
-      max-height: 106px;
-      max-width: 144px
-    }
+    max-height: 106px;
+    max-width: 144px;
 
     .ms-DocumentCardPreview-icon {
       .ms-Image-image {
-        max-height: 16px;
-        max-width: 16px;
+        max-height: 32px;
+        max-width: 32px;
       }
     }
   }
 
+  .ms-DocumentCardPreview.is-fileList {
+    @include border-right(1px, solid, $ms-color-neutralLight);
+  }
+
+  .ms-DocumentCardPreview-fileList {
+    li {
```

Comment:
this is a really specific rule. it might be better to avoid elements, but up to you.